login.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { 
+  AppRegistry, 
+  Text,
+  StyleSheet,
+  View,
+  TextInput,
+  Button,
+  Alert,
+  Image,
+} from 'react-native';
+import { Actions, Router, Scene } from 'react-native-router-flux'
+
+import Home from './Home';
```

Comment:
Nope, that is simply using the Actions class from react-native-router-flux, which does bring you to the Home page, because that it how you defined your Scenes in main.js. But you do not need to actually import Home here, because you never use that code. You import Home in main.js, because that is where it is used